frontend/packages/console-app/src/components/user-preferences/__tests__/UserPreferenceDropdownField.spec.tsx

```diff
@@ -38,7 +38,7 @@ describe('UserPreferenceDropdownField', () => {
     mockUserSettings.mockReturnValue(['value', () => {}, true]);
     wrapper = shallow(<UserPreferenceDropdownField {...props} />);
     expect(wrapper.find('[data-test="dropdown id"]').exists()).toBeTruthy();
-    expect(wrapper.find(SelectDeprecated).props().selections).toBe('label');
+    expect((wrapper.find(Select).props() as { selected: string }).selected).toBe('label');
```

Comment:
Rather than redefining the prop type, I think we want something like the folllowing:
```suggestion
    expect((wrapper.find(Select).props() as SelectProps).selected).toBe('label');
```
See [existing example](https://github.com/openshift/console/blob/684988c9fa7a69e827292a35a87b019f70984ae6/frontend/packages/dev-console/src/components/deployments/__tests__/EditDeployment.spec.tsx#L37).